notebooks/203-meter-reader/203-meter-reader-digital.ipynb

```diff
@@ -0,0 +1,1055 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #96.    text_decoder = RecLabelDecode(character_dict_path="../data/text/ppocr_keys_v1.txt",
</pre><p>Missing file: ppocr_keys_v1.txt</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1030/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
The most thing about paddleocr will be very similar to https://github.com/openvinotoolkit/openvino_notebooks/blob/2023.2/notebooks/405-paddle-ocr-webcam/405-paddle-ocr-webcam.ipynb, and the `ppocr_keys_v1.txt` file can be found in the `ipynb`.

The most related codes are:

```
# Download font and a character dictionary for printing OCR results.
font_path = utils.download_file(
    url='https://raw.githubusercontent.com/Halfish/lstm-ctc-ocr/master/fonts/simfang.ttf',
    directory='fonts'
)
character_dictionary_path = utils.download_file(
    url='https://raw.githubusercontent.com/WenmuZhou/PytorchOCR/master/torchocr/datasets/alphabets/ppocr_keys_v1.txt',
    directory='fonts'
)
```

And you can download the file from that url :)